### PR TITLE
Show filter results

### PR DESCRIPTION
**Pull request by moonso**
- Adress #68 
- Special one for you @vasiliosz :dancer:

### CODE REVIEW

**Comment by moonso** [comment]
Backend fixed, just need to show result in frontend

**Comment by guillermo-carrasco** [comment]
haha awesome! I was going to ask: where is this shown in the frontend? :-P

The backend looks good to me ^^

**Comment by moonso** [comment]
Soon @guillermo-carrasco soon

**Comment by moonso** [comment]
Ok @guillermo-carrasco , @vasiliosz Iäve added a small message that tells the user how many variants that passed the filter if it is below 1000

**Comment by moonso** [comment]
Do you think that 1000 variants is a good number?

File: puzzle/plugins/vcf/mixins/variant_mixin.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def variants(self, case_id, skip=0, count=1000, filters=None):
             else:
                 skip_index += 1
         
-        return Results(iter(result), len(result))
+        return Results(result, len(result))
```
Comment (guillermo-carrasco) [inline puzzle/plugins/vcf/mixins/variant_mixin.py]: any reason to change this?

Comment (moonso) [inline puzzle/plugins/vcf/mixins/variant_mixin.py]: Yes it was stupid to convert the list to a iterator in the first place, we already store it in memory

Comment (guillermo-carrasco) [inline puzzle/plugins/vcf/mixins/variant_mixin.py]: I'll give you that :sweat_smile:

**Comment by vasiliosz** [comment]
Wow you guys are really on fire today. I can check in a week :/ But nr of variants shown is controlled by the js setting, should be easy to find

**Comment by guillermo-carrasco** [comment]
@moonso done, check out the last commit in this branch :-)

**Comment by moonso** [comment]
Fantastic @guillermo-carrasco !!

**Pull request merged**